src/kernels/composable_kernel/include/utility/amd_buffer_addressing.hpp

```diff
@@ -250,15 +194,11 @@ __device__ half_t amd_buffer_load<half_t, 1>(const half_t* p_src_block,
     // fill in byte 3
     src_block_config.range[3] = 0x00027000;
 
-#if !CK_WORKAROUND_SWDEV_231101
     index_t src_thread_addr_offset = src_thread_data_offset * sizeof(half_t);
     index_t src_const_addr_offset  = src_const_data_offset * sizeof(half_t);
 
-    return __llvm_amdgcn_buffer_load_f16(
-        src_block_config.data, 0, src_thread_addr_offset + src_const_addr_offset, false, false);
-#else
-    return p_src_block[src_thread_data_offset + src_const_data_offset];
-#endif
+    return __llvm_amdgcn_raw_buffer_load_f16(
```

Comment:
@asroy perhaps it's better the add a comment here saying Soffset is currently hard-coded as 0.

Reply:
fixed